sdk/master_board_sdk/CMakeLists.txt

```diff
@@ -66,89 +61,54 @@ ADD_LIBRARY(${PROJECT_NAME} SHARED
-  ENDIF(BUILD_WITH_COMMIT_VERSION)
-  ADD_HEADER_GROUP(${PYWRAP}_HEADERS)
-  ADD_SOURCE_GROUP(${PYWRAP}_SOURCES)
-
   TARGET_INCLUDE_DIRECTORIES(${PYWRAP} SYSTEM PRIVATE ${PYTHON_INCLUDE_DIR})
   TARGET_LINK_LIBRARIES(${PYWRAP} ${PROJECT_NAME})
   TARGET_LINK_BOOST_PYTHON(${PYWRAP})
-  SET_TARGET_PROPERTIES(${PYWRAP} PROPERTIES INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
-
+  SET_TARGET_PROPERTIES(${PYWRAP} PROPERTIES
+    INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
   INSTALL(TARGETS ${PYWRAP} DESTINATION ${${PYWRAP}_INSTALL_DIR})
-  # --- PACKAGING --- #
-  # Format string
-  SET(_PKG_CONFIG_PYWRAP_LIBDIR ${${PYWRAP}_INSTALL_DIR})
-  SET(_PKG_CONFIG_PYWRAP_BINDIR ${${PYWRAP}_INSTALL_DIR})
-  SET(_PKG_CONFIG_PYWRAP_CONFLICTS)
-  SET(_PKG_CONFIG_PYWRAP_REQUIRES "${PROJECT_NAME}")
-  FOREACH(dep ${PKG_CONFIG_PYWRAP_REQUIRES})
-    SET(_PKG_CONFIG_PYWRAP_REQUIRES "${_PKG_CONFIG_PYWRAP_REQUIRES}, ${dep}")
-  ENDFOREACH(dep ${PKG_CONFIG_PYWRAP_REQUIRES})
-  SET(_PKG_CONFIG_PYWRAP_LIBS "-L\${libdir} -l${PYWRAP}")
-  IF(APPLE)
-    SET(_PKG_CONFIG_PYWRAP_LIBS "${_PKG_CONFIG_PYWRAP_LIBS} -Wl,-undefined,dynamic_lookup,${Boost_${UPPERCOMPONENT}_LIBRARY}")
-  ELSE(APPLE)
-    SET(_PKG_CONFIG_PYWRAP_LIBS "${_PKG_CONFIG_PYWRAP_LIBS} ${LIBINCL_KW}boost_python")
-  ENDIF(APPLE)
-  SET(_PKG_CONFIG_PYWRAP_CFLAGS "-I\${includedir} -I/usr/local/include")
-  SET(_PKG_CONFIG_PYWRAP_CFLAGS "${_PKG_CONFIG_PYWRAP_CFLAGS} -I${PYTHON_INCLUDE_DIRS}")
-  FOREACH(cflags ${CFLAGS_DEPENDENCIES})
-    SET(_PKG_CONFIG_PYWRAP_CFLAGS "${_PKG_CONFIG_PYWRAP_CFLAGS} ${cflags}")
-  ENDFOREACH(cflags ${CFLAGS_DEPENDENCIES})
-  CONFIGURE_FILE(
-     "${CMAKE_CURRENT_SOURCE_DIR}/srcpy/master-board-sdk-py.pc.cmake"
-     "${CMAKE_CURRENT_BINARY_DIR}/srcpy/master-board-sdk-py.pc")
-
-  INSTALL(
-      FILES "${CMAKE_CURRENT_BINARY_DIR}/srcpy/master-board-sdk-py.pc"
-      DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
-      PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE)
+  
+  # --- Allow to do: make python --- #
+  ADD_CUSTOM_TARGET(python)
+  SET_TARGET_PROPERTIES(python PROPERTIES EXCLUDE_FROM_DEFAULT_BUILD True)
+  ADD_DEPENDENCIES(python ${PYWRAP})
 
+  # --- Pre-build the python files for efficiency --- #
   PYTHON_BUILD(example example.py)
   PYTHON_BUILD(example listener.py)
   PYTHON_BUILD(example com_analyser.py)
 ENDIF(BUILD_PYTHON_INTERFACE)
 
 
 # --- CHECK EXAMPLES -----------------------------------------------------------
-ADD_EXECUTABLE(master-board-example example/example.cpp)
-TARGET_LINK_LIBRARIES(master-board-example ${PROJECT_NAME})
-INSTALL(TARGETS master-board-example DESTINATION bin)
+ADD_EXECUTABLE(master_board_example example/example.cpp)
+TARGET_LINK_LIBRARIES(master_board_example ${PROJECT_NAME})
+INSTALL(TARGETS master_board_example DESTINATION bin)
 
 ADD_EXECUTABLE(master-board-example-pd example/example_pd.cpp)
 TARGET_LINK_LIBRARIES(master-board-example-pd ${PROJECT_NAME})
 INSTALL(TARGETS master-board-example-pd DESTINATION bin)
```

Comment:
I guess these new targets should be changed to lower-case like `master_board_example_pd` as well?